llvm/include/llvm/DebugInfo/DWARF/DWARFDebugInfoEntry.h

```diff
@@ -44,6 +44,10 @@ class DWARFDebugInfoEntry {
 
   uint64_t getOffset() const { return Offset; }
 
+  void setAbbrevDecl(const DWARFAbbreviationDeclaration *AbbrDecl) {
```

Comment:
This is cosmetic, but it might be better to add this as a default = nullptr argument in the constructor, to make it clear that this is not supposed to be modified after creation.

Reply:
Great suggestion!